pom.xml

```diff
@@ -50,9 +50,23 @@
     <nativeLibOnlyDir>${project.build.directory}/native-lib-only</nativeLibOnlyDir>
     <nativeJarWorkdir>${project.build.directory}/native-jar-work</nativeJarWorkdir>
     <aprVersion>1.5.2</aprVersion>
+    <aprMd5>98492e965963f852ab29f9e61b2ad700</aprMd5>
```

Comment:
apache publishes md5 checksums. It would be a bit more secure but more process to instead have a maintainer verify the md5 of the archive but compute a sha256 to use here. wdyt?